.github/workflows/packages.yml

```diff
@@ -27,16 +27,17 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, windows-latest, macos-latest]
-        pyver: [cp37, cp38, cp39, cp310]
+        pyver: [cp37, cp38, cp39, cp310, cp311]
 
     steps:
       - name: Checkout repos
         uses: actions/checkout@v3
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.8.1
+        uses: pypa/cibuildwheel@v2.11.1
         env:
           CIBW_BUILD: ${{matrix.pyver}}-*
+          CIBW_PRERELEASE_PYTHONS: True
```

Comment:
We don't need a prerelease Python anymore, right?

Reply:
I've been leaving it in personally, but it can certainly be removed.  I figure I'll want it when adding 3.12 support and in the interrim it will most likely have no effect.

Feel free to commit this if you want it removed.

```suggestion
```

Reply:
It is [stated in pretty clear terms](https://cibuildwheel.readthedocs.io/en/stable/options/#prerelease-pythons) that distributing pre-release packages is a bad idea.

There are other occurrences in this changeset: can you please provide a changeset instead of a suggestion? Thank you :)

Reply:
Removed in https://github.com/dvarrazzo/py-setproctitle/pull/120/commits/8e70374c4d9d2b6f34ee1056068bd077812461eb.